qa-tests-backend/src/test/groovy/ReconciliationTest.groovy

```diff
@@ -183,14 +183,14 @@ class ReconciliationTest extends BaseSpecification {
 
         // Recreate sensor
         try {
-            orchestrator.createOrchestratorDeployment(sensorOrchestratorDeployment)
+            orchestrator.scaleDeployment("stackrox", "sensor", 1)
         } catch (Exception e) {
             log.error("Error re-creating the sensor: ", e)
             throw e
         }
         Services.waitForDeployment(sensorDeployment)
 
-        def maxWaitForSync = 100
+        def maxWaitForSync = 200
```

Comment:
I think using `while (t.IsValid()) {` when we call `Services` could cause problems when network is down consider using a `Helpers.evaluateWithRetry` as it will handle exceptions

Reply:
Done.